front/src/modules/simulationResult/SimulationResultExport/SimulationReportSheetScenario.tsx

```diff
@@ -0,0 +1,328 @@
+import { Table, TR, TH, TD } from '@ag-media/react-pdf-table';
+import { Page, Text, Image, Document, View, Link } from '@react-pdf/renderer';
+import { useTranslation } from 'react-i18next';
+
+import styles from 'applications/stdcm/components/SimulationReportStyleSheet';
+import { getStopDurationTime } from 'applications/stdcm/utils/formatSimulationReportSheet';
+import logoSNCF from 'assets/simulationReportSheet/logo_sncf_reseau.png';
+import type {
+  PathfindingResultSuccess,
+  RollingStockWithLiveries,
+  ScenarioResponse,
+} from 'common/api/osrdEditoastApi';
+import type { OperationalPointWithTimeAndSpeed } from 'modules/trainschedule/components/DriverTrainSchedule/types';
+import { formatDateToString } from 'utils/date';
+import { secToHoursString } from 'utils/timeManipulation';
+
+export type SimulationSheetData = {
+  trainName: string | undefined;
+  rollingStock: RollingStockWithLiveries;
+  speedLimitByTag: string;
+  departure_time: string;
+  creationDate: Date;
+};
+export type SimulationReportSheetScenarioProps = {
+  path: PathfindingResultSuccess;
+  scenarioResponse: ScenarioResponse | undefined;
+  scenarioData: SimulationSheetData;
+  mapCanvas?: string;
+  operationalPointsList: OperationalPointWithTimeAndSpeed[];
+};
+
+const SimulationReportSheetScenario = ({
+  path,
+  scenarioResponse,
+  scenarioData,
+  mapCanvas,
+  operationalPointsList,
+}: SimulationReportSheetScenarioProps) => {
+  const { t } = useTranslation([
+    'stdcm-simulation-report-sheet',
+    'stdcm',
+    'operationalStudies/study',
+  ]);
+  let renderedIndex = 0;
+
+  const { rollingStock, speedLimitByTag, creationDate } = scenarioData;
+
+  const headerTitle = t('operationalStudies/study:simulationSheet');
+
+  return (
+    <Document>
+      <Page wrap={false} style={styles.main.page} size={[1344]}>
+        <View style={styles.header.numberDateBanner}>
+          <View style={styles.header.stdcmTitleBox}>
+            <View style={styles.header.stdcm}>
+              <Text style={styles.header.title}>{headerTitle}</Text>
+            </View>
+          </View>
+          <View style={styles.header.numericInfo}>
+            <Text style={styles.header.number}>{scenarioData.trainName}</Text>
+          </View>
+          <View style={styles.header.numericInfo}>
+            <Text style={styles.header.number}>
+              {t('operationalStudies/study:scenarioWithTwoPoints')}
+              {scenarioResponse?.name}
```

Comment:
the suspension points extend beyond the square when title is too long

Reply:
Done